bin/martinize2

```diff
@@ -1067,13 +1069,13 @@ def entry():
 
     # apply idr specific bonded potentials if required
     if args.idr_tune:
-        if args.to_ff == 'martini3001':
-            vermouth.processors.IDRBonds([(int(start), int(stop)) for start, stop in args.water_idrs]
-                                         ).run_system(system)
+        if not target_ff.has_feature("idr"):
+            LOGGER.warning('Improved IDR potentials are not implemented '
+                           'for this force field',
+                           type="missing-feature")
         else:
-            raise argparse.ArgumentError(idr_tuning,
-                                         message=("You are trying to apply IDR biasing to a force field which "
-                                                  "isn't martini3001, which this is not designed for."))
+            vermouth.IDRBonds(idr_regions = [(int(start), int(stop)) for start, stop in args.water_idrs]).run_system(system)
+            vermouth.DoLinks().run_system(system)
```

Comment:
I don't think you actually want to run DoLinks a second time. Instead, move the IDRBonds annotator to just after the mapping